src/identity/selectors/index.ts

```diff
@@ -2,12 +2,24 @@
 import {AppState} from 'src/types'
 import {RemoteDataState} from '@influxdata/clockface'
 
+export const selectCurrentAccount = (
+  state: AppState
+): AppState['identity']['currentIdentity']['account'] => {
+  return state.identity.currentIdentity.account
+}
+
 export const selectCurrentAccountId = (
   state: AppState
 ): AppState['identity']['currentIdentity']['account']['id'] => {
   return state.identity.currentIdentity.account.id
 }
 
+export const selectCurrentAccountName = (
+  state: AppState
+): AppState['identity']['currentIdentity']['account']['name'] => {
+  return state.identity.currentIdentity.account.name
```

Comment:
Good point - I'll remove. Generally, I've tried to get away from selectors that return an entire object rather than a primitive value. However, in this case, the entire account object will never change, because the current account remains identical until app reload.